docker-compose.yml

```diff
@@ -14,6 +14,7 @@ services:
       MICROSERVICE_TOKEN: ${microservice_token}
       MONGODB_HOST: mongo
       MONGODB_PORT: 27017
+      ENV: local
```

Comment:
Is this present in other instances as well, if not how does that affect the sentry logging

Reply:
This is only for the locally running version and doesn't define the containers for anything AWS hosted